src/main/java/gov/cabinetofice/gapuserservice/service/JwtBlacklistService.java

```diff
@@ -0,0 +1,33 @@
+package gov.cabinetofice.gapuserservice.service;
+
+import gov.cabinetofice.gapuserservice.model.JwtBlacklist;
+import gov.cabinetofice.gapuserservice.repository.JwtBlacklistRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.Clock;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+@RequiredArgsConstructor
+@Service
+public class JwtBlacklistService {
+    private final JwtBlacklistRepository jwtBlacklistRepository;
+
+    private final Clock clock;
+
+    public void addJwtToBlacklist(final String jwt) {
+        JwtBlacklist blacklist = JwtBlacklist.builder()
+                .jwt(jwt)
+                .build();
+        this.jwtBlacklistRepository.save(blacklist);
+    }
+
+    public boolean isJwtInBlacklist(final String jwt) {
+        return jwtBlacklistRepository.existsByJwtIs(jwt);
+    }
+
+    public Long deleteExpiredJwts() {
+        return this.jwtBlacklistRepository.deleteByExpiryDateLessThan(new Date(ZonedDateTime.now(clock).toInstant().toEpochMilli()));
```

Comment:
This doesn't need the `new` keyword, you can make static calls to the `ZonedDateTime` class :)